digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/package.scala

```diff
@@ -39,7 +39,7 @@ package object viite {
   /* Maximum distance of regular road link geometry to suravage geometry difference where splitting is allowed */
   val MaxSuravageToleranceToGeometry = 0.5
 
-  val ErrorNoMatchingProjectLinkForSplit = "Suravage-linkkiä vastaavaa käsittelemätöntä tieosoitelinkkiä ei löytynyt projektista"
+  val ErrorNoMatchingProjectLinkForSplit = "Suravage-linkkiä vastaavaa käsittelemätöntä tieosoite linkkiä ei löytynyt projektista"
```

Comment:
Original was correctly spelled o.O

Reply:
My bad :/ The google suggested it ! I'll revert the change